### PR TITLE
Remove unused package code

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

This takes one more step towards dropping kanso by deleting or moving some of the code in the kujua-sms and kujua-utils packages. It introduces a little duplication but this can be resolved when we have a solution for shared libraries.

medic/medic-webapp#3019 (partial)

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

### CODE REVIEW

File: lib/views.js [hunk 1]
```diff
@@ -83,19 +83,6 @@ exports.due_tasks = {
   }
 };
 
-exports.duplicate_form_submissions = {
```
Comment (SCdF) [inline lib/views.js]: Nice, one more down!

File: static/js/bootstrapper.js [hunk 2]
```diff
@@ -83,6 +81,22 @@ var utils = require('kujua-utils');
     return callback(err);
   };
 
+  var hasRole = function(userCtx, role) {
```
Comment (SCdF) [inline static/js/bootstrapper.js]: Do you want to note where you've duplicated this code in the code itself, so we don't forget what to pull out once we have a solution to this?

File: static/js/modules/set-task-state.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+module.exports = function(task, state) {
```
Comment (SCdF) [inline static/js/modules/set-task-state.js]: I believe this is another example of copied code

File: static/js/bootstrapper.js (hunk 2 above)
Comment (garethbowen) [reply static/js/bootstrapper.js]: Added comments everywhere this is used.

File: static/js/modules/set-task-state.js (hunk 3 above)
Comment (garethbowen) [reply static/js/modules/set-task-state.js]: Added comments everywhere this is used.

**Pull request merged**